mindsdb/integrations/handlers/anthropic_handler/README.md

```diff
@@ -0,0 +1,24 @@
+# Anthropic Handler
+Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can train your existing text data with Anthropic LLM
```

Comment:
> You can train your existing text data with Anthropic LLM

This doesn't seem accurate. This handler does not support fine-tuning (i.e. no `update()` method), so I would suggest rewording to something like:

> The Anthropic ML handler integrates Anthropic LLMs with MindsDB. You can use it to generate text completions with the popular Claude LLM family for your existing text data.

Reply:
Thanks for the review, I have made the necessary changes